src/draw/lv_draw_label.c

```diff
@@ -563,74 +566,81 @@ static uint8_t hex_char_to_num(char hex)
 void lv_draw_unit_draw_letter(lv_draw_task_t * t, lv_draw_glyph_dsc_t * dsc,  const lv_point_t * pos,
                               const lv_font_t * font, uint32_t letter, lv_draw_glyph_cb_t cb)
 {
-    lv_font_glyph_dsc_t g;
-
     if(lv_text_is_marker(letter)) /*Markers are valid letters but should not be rendered.*/
         return;
 
     LV_PROFILER_DRAW_BEGIN;
-    bool g_ret = lv_font_get_glyph_dsc(font, &g, letter, '\0');
-    if(g_ret == false) {
-        /*Add warning if the dsc is not found*/
-        LV_LOG_WARN("lv_draw_letter: glyph dsc. not found for U+%" LV_PRIX32, letter);
+    lv_font_glyph_dsc_t local_glyph;
+    lv_font_glyph_dsc_t * g = dsc->g;
+    if(g == NULL) {
+        /*If the glyph dsc is not set then get it from the font*/
+        dsc->g = g = &local_glyph;
+        bool g_ret = lv_font_get_glyph_dsc(font, g, letter, '\0');
+        if(g_ret == false) {
+            /*Add warning if the dsc is not found*/
+            LV_LOG_WARN("lv_draw_letter: glyph dsc. not found for U+%" LV_PRIX32, letter);
+        }
     }
 
```

Comment:
to make the change to be smallest.
and reduce mem access from far mem address.

```suggestion
lv_font_glyph_dsc_t g = *dsc->g;
```